src/pages/Result.vue

```diff
@@ -36,6 +45,11 @@ export default {
             })
         })
         return tags
+      },
+      restart() {
+        if(confirm("Are you sure you want to start again? The current results will be lost")){
```

Comment:
We shouldn't be using the native `confirm()` dialog, we can package this up as a modal Vuetify dialog, checkout https://vuetifyjs.com/en/components/dialogs and https://www.npmjs.com/package/vuetify-confirm